ts/packages/node/src/index.ts

```diff
@@ -6,8 +6,9 @@ export { NoopConfigStore, InMemoryConfigStore } from '@configu/ts';
 
 export * from './stores/Configu';
 export * from './stores/JsonFile';
+export * from './stores/IniFile';
 
-export * from './stores/AWSParameterStore'
+export * from './stores/AWSParameterStore';
```

Comment:
Please update your branch from main, it will resolve this 🙏🏻

Reply:
@RichardAkman Could you have me rebase it? I tried myself but reverted because I wasn't completely sure that I will not mess things up.

Reply:
Please merge the incoming changes to `package.json` and append your own changes after that. Once the `package.json` is sorted, delete the `package-lock.json` and run `npm install` to generate an updated version of the lock file. Should resolve the issue.